src/modules/tools/zprobe/CartGridStrategy.cpp

```diff
@@ -396,11 +470,32 @@ float CartGridStrategy::findBed()
 bool CartGridStrategy::doProbe(Gcode *gc)
 {
     gc->stream->printf("Rectangular Grid Probe...\n");
+
+    if(only_by_two_corners){
+        if(gc->has_letter('X') && gc->has_letter('X') && gc->has_letter('A') && gc->has_letter('B')){
+            x_start = gc->get_value('X'); // override default probe start point, will get saved
+            y_start = gc->get_value('Y'); // override default probe start point, will get saved
+            x_size = gc->get_value('A'); // override default probe width, will get saved
+            y_size = gc->get_value('B'); // override default probe length, will get saved
```

Comment:
From what I understood of the documentation, isn't A,B defined as the size of the rectangle? This is being used as the coordinates of the other two points of the rectangle (which will probably cause some confusion). Either fix the documentation or change the implementation to reflect the x_size and y_size being relative coordinates from the start points.